gatsby-node.js

```diff
@@ -43,8 +87,6 @@ const transformCodeinjection = (posts) => {
             post.codeinjection_styles += headInjection.styles;
         }
 
-        post.codeinjection_styles = _.isNil(post.codeinjection_styles) ? '' : post.codeinjection_styles;
```

Comment:
This will result in possible breaking change - previously this field would result in empty string (if _.isNil check is truthy), with removal this will result in null. This can cause problems if someone tries to some string manipulation assuming that this field is never null

Reply:
On review, I remember why I removed this check. If there is no code injection in either the head or foot, the null check above on lines 78-80 will return null rather than an empty string. This means it is possible for `codeinjection_styles` to be null, that clients can already expect to deal with it, and so might not be a breaking change. Let me know what you think.